srcpkgs/smpq/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'smpq'
+pkgname=smpq
+version=1.6
+revision=1
+build_style=cmake
+configure_args="-DWITH_KDE=OFF"
+makedepends="bzip2-devel zlib-devel StormLib"
+short_desc="StormLib MPQ archiving utility"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://launchpad.net/smpq"
+distfiles="https://launchpad.net/smpq/trunk/${version}/+download/smpq_${version}.orig.tar.gz"
+checksum=b5d2dc8a5de8629b71ee5d3612b6e84d88418b86c5cd39ba315e9eb0462f18cb
+patch_args="-Np1"
```

Comment:
This is the default, its not necessary.

Reply:
In which line, what is it not necessary?

Reply:
The `patch_args`

Reply:
Done!